config.json

```diff
@@ -15,7 +15,7 @@
     {
       "uuid": "57b76aba-a485-464d-84ba-e9a445b25be6",
       "slug": "bob",
-      "core": false,
+      "core": true,
```

Comment:
I think I'd like to see bob moved down a bit as I think it is harder than `sum-of-multiples` and `leap`